contrib/IECoreArnold/src/IECoreArnold/ParameterAlgo.cpp

```diff
@@ -92,6 +92,16 @@ void setParameterInternal( AtNode *node, AtString name, int parameterType, bool
 					AiNodeSetInt( node, name, data->readable() );
 				}
 				break;
+			case AI_TYPE_UINT :
+				if( const IntData *data = runTimeCast<const IntData>( value ) )
+				{
+					AiNodeSetUInt( node, name, data->readable() );
```

Comment:
Should we concerned about passing an int in to the UInt function ? Should we guard against negative numbers before calling this?